src/main/java/fi/metatavu/metamind/bot/MetamindBotConfiguration.java

```diff
@@ -148,7 +150,17 @@ public String getStoryJson() {
 
   @Override
   public List<GlobalIntent> getGlobalIntents() {
-    return Collections.emptyList();
+    Map<String, String> globalIntentMap = config.getGlobalIntents();
+    if (globalIntentMap == null) {
+      return Collections.emptyList();
+    }
+    
+    List<GlobalIntent> globalIntents = globalIntentMap.entrySet()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "globalIntents". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1488)